### PR TITLE
Create Themable Themes (in CSS not TS)

### PR DESCRIPTION
**Pull request by JustifydWarrior**
* move themes out of inline styles and into (s)css
* simplify dropdown to use a container class rather than app state & ts to hold styles
* make editing & adding themes very simple: 
  * copypasta some existing theme & rename
  * add entry to select box with class name as value   
  
 live at [bitclout.pixelbypixelstudios.live](https://bitclout.pixelbypixelstudios.live)

### CODE REVIEW

**Comment by teambrixmusic** [comment]
@JustifydWarrior This is really cool

**Comment by maebeam** [comment]
This is great! If you'd like to update this PR I'd love to get it merged 💪🏼

**Comment by JustifydWarrior** [comment]
Sorry, I've been neck deep implementing Twitch Drops for our game. This should be up to date again.

My employer pushed pause, but not stop, on running a public frontend for our node, which is why it's been pretty quiet on my end.

File: src/app/theme/themes/legends.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Ledgends Theme
+// By:
+.ledgends[app-theme] {
```
Comment (lazynina) [inline src/app/theme/themes/legends.scss:3]:
```suggestion
.legends[app-theme] {
```

File: src/app/theme/themes/legends.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
+// Ledgends Theme
```
Comment (lazynina) [inline src/app/theme/themes/legends.scss:1]:
```suggestion
// Legends Theme
```

File: src/app/theme/themes/_index.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
+@import 'light.scss';
+@import 'dark.scss';
+@import 'icydark.scss';
+@import 'legends.scss';
+@import 'cake.scss';
+@import 'greenish.scss';
```
Comment (lazynina) [inline src/app/theme/themes/_index.scss:6]: We need to copy over the cake and greenish themes into their own scss files.

**Review by lazynina — CHANGES_REQUESTED** [review]

**Comment by lazynina** [comment]
also need to add the `--button` property

File: src/app/theme/themes/_index.scss (hunk 3 above)
Comment (JustifydWarrior) [reply src/app/theme/themes/_index.scss:6]: that was a big miss, just forgot to commit them.

File: src/app/theme/themes/legends.scss (hunk 2 above)
Comment (JustifydWarrior) [reply src/app/theme/themes/legends.scss]: oops we're not looking for "ledge ends"

**Comment by JustifydWarrior** [comment]
should be all set. nice catch on the buttons. they got added and caused a regression in the older scss.

**Comment by lazynina** [comment]
The themes are working now. However, this PR causes the page to refresh when a new theme is selected whereas the current implementation does not reload the page. Is there a way to accomplish the theme change without a reload here?

**Comment by JustifydWarrior** [comment]
yes there is.

**Comment by JustifydWarrior** [comment]
@lazynina it's working as expected now. I tested side by side with the dot com site several times.

**Review by lazynina — APPROVED** [review]
looks good to me! @maebeam for second review

**Comment by teambrixmusic** [comment]
Awesome! So exciting!

**Review by maebeam — APPROVED** [review]
Thank you so much for cleaning this up!

**Pull request merged**